release-calendar/src/client/models/release-event.ts

```diff
@@ -15,19 +15,20 @@
  */
 
 import {Channel} from '../../types';
-import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
-import {ReleaseEventInput} from './view-models';
+import {EventInput} from '@fullcalendar/core/structs/event';
+import {ExtendedEventSourceInput} from '@fullcalendar/core/structs/event-source';
```

Comment:
If `ExtendedEventSourceInput` isn't required, I would opt for using the smaller, more simple `EventInput`.

Reply:
In order to use the get methods, I have to have it as ExtendedEventSourceInput.
```
allReleases.get(promotion.channel) as ExtendedEventSourceInput)
          .events
```
This wouldn't work in `getSingelReleaseEvents`

Reply:
Update: adjustment was made to getSingleReleaseEvents that removed the need for the events on the EventSourceInput to be accessed.